### PR TITLE
New document creation added with UI fixes

### PR DESCRIPTION
**Pull request by 2Dsharp**
Ready for merge conflicts. Some UI designs have been fixed along with addition of the open new dialog. Further validation with new documents necessary.

### CODE REVIEW

**Comment by 2Dsharp** [comment]
Conflicts have been resolved.

File: src/widgets/paintwidget.h [hunk 1]
```diff
@@ -9,28 +9,31 @@
...
     friend class Drawing;
     Q_OBJECT
 public:
     PaintWidget(const QString &imagePath, QWidget *parent=0);
-    PaintWidget(const QSize &imageSize, QWidget *parent=0);
+    PaintWidget(const Canvas *document, QWidget *parent=0);
 
     void setImagePath(QString path);
-    QString imagePath() const;
+    QString getImagePath() const;
```
Comment (hellozee) [inline src/widgets/paintwidget.h]: Inline those functions

File: src/model/canvas.h [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        PIXELS, INCHES, CENTIMETERS
+    };
+    enum ResolutionUnit
+    {
+        PPI, DPI
+    };
+
+    Canvas(QString docName,
+             int width,
+             int height,
```
Comment (hellozee) [inline src/model/canvas.h]: put width and height on the same line

File: src/widgets/paintwidget.h (hunk 1 above)
Comment (2Dsharp) [reply src/widgets/paintwidget.h]: Already done.

File: src/model/canvas.h (hunk 2 above)
Comment (2Dsharp) [reply src/model/canvas.h]: Done.

**Comment by 2Dsharp** [comment]
Conflicts resolved, check again.

**Pull request merged**